src/test/javascript/spec/component/programming-exercise/update-components/programming-exercise-information.component.spec.ts

```diff
@@ -65,8 +70,18 @@ describe('ProgrammingExerciseInformationComponent', () => {
 
     it('should should calculate Form Sections correctly', () => {
         const calculateFormValidSpy = jest.spyOn(comp, 'calculateFormValid');
-        const editableField = { editingInput: { valueChanges: new Subject(), valid: true } } as any as TableEditableFieldComponent;
-        comp.exerciseTitleChannelComponent = { titleChannelNameComponent: { formValidChanges: new Subject(), formValid: true } } as ExerciseTitleChannelNameComponent;
+        const editableField = {
+            editingInput: {
+                valueChanges: new Subject(),
+                valid: true,
+            },
+        } as any as TableEditableFieldComponent;
```

Comment:
why is the any needed?

Reply:
![image](https://github.com/user-attachments/assets/945cf72a-cc15-49fb-93d1-8e9d3b390540)